src/DerConverter/Asn/DerAsnUtcTime.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DerConverter.Asn
+{
+    public class DerAsnUtcTime : DerAsnType
+    {
+        private readonly List<byte> _bytes = new List<byte>();
+
+        internal DerAsnUtcTime(Queue<byte> rawData)
+            : base(DerAsnTypeTag.UtcTime)
+        {
+            if (rawData == null) throw new ArgumentNullException(nameof(rawData));
+            _bytes.AddRange(rawData.DequeueAll());
+        }
+
+        public DerAsnUtcTime(DateTime time)
```

Comment:
Should take a `DateTimeOffset` + a bool `asUtc` + a bool `includeSeconds`, so it supports all flavors as described in https://www.obj-sys.com/asn1tutorial/node15.html